universal-login-commons/src/core/models/wallets/SerializableConfirmedMigratingWallet.ts

```diff
@@ -0,0 +1,17 @@
+import {asString, asObject} from '@restless/sanitizers';
+
+export interface SerializableConfirmedMigratingWallet {
```

Comment:
I think we should extend ConfirmedWallet here. What do you think?

Reply:
We could but it is more readable this way.